src/Payum/Core/Bridge/Propel2/Resources/config/propel.xml.dist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
```

Comment:
Is it really worth it? You can find this file in the propel doc http://propelorm.org/documentation/02-buildtime.html or
here http://propelorm.org/documentation/10-configuration.html

What if they change something in schema we have to do same in this file and this the main reason not keep it here.